templates/admin/user/edit.tmpl

```diff
@@ -70,6 +70,21 @@
 					<input id="password" name="password" type="password" autocomplete="new-password">
 					<p class="help">{{ctx.Locale.Tr "admin.users.password_helper"}}</p>
 				</div>
+
+				<div class="field {{if .Err_Language}}error{{end}}">
+					<label for="language">{{ctx.Locale.Tr "settings.language"}}</label>
+					<div class="ui selection dropdown">
+						<input name="language" type="hidden" value="{{.User.Language}}">
+						{{svg "octicon-triangle-down" 14 "dropdown icon"}}
+						<div class="text">{{range .AllLangs}}{{if eq $.User.Language .Lang}}{{.Name}}{{end}}{{end}}</div>
+						<div class="menu">
+						{{range .AllLangs}}
+							<div class="item{{if eq $.User.Language .Lang}} active selected{{end}}" data-value="{{.Lang}}">{{.Name}}</div>
+						{{end}}
+						</div>
+					</div>
```

Comment:
Can't we extract a common template with the site footer?

Reply:
its similar but differ

Reply:
```diff
diff --git a/user_settings_appearance.tmpl b/admin_user_edit.tmpl
index 3995b58..3a421d1 100644
--- a/user_settings_appearance.tmpl
+++ b/admin_user_edit.tmpl
@@ -1,12 +1,8 @@
-<div class="field">
-       <div class="ui language selection dropdown" id="language">
-               <input name="language" type="hidden" value="{{.SignedUser.Language}}">
-               {{svg "octicon-triangle-down" 14 "dropdown icon"}}
-               <div class="text">{{range .AllLangs}}{{if eq $.SignedUser.Language .Lang}}{{.Name}}{{end}}{{end}}</div>
-               <div class="menu">
-                       {{range .AllLangs}}
-                               <div class="item{{if eq $.SignedUser.Language .Lang}} active selected{{end}}" data-value="{{.Lang}}">{{.Name}}</div>
-                       {{end}}
-               </div>
+{{if .User.Language}}
+       <div class="flex-item-body">
+               <span class="flex-text-inline">
+                       <b>{{ctx.Locale.Tr "settings.language"}}:</b>
+                       {{range .AllLangs}}{{if eq $.User.Language .Lang}}{{.Name}}{{end}}{{end}}
+               </span>
        </div>
-</div>
+{{end}}
```
```diff
diff --git a/admin_user_view_details.tmpl b/user_settings_appearance.tmpl
index 3a421d1..3995b58 100644
--- a/admin_user_view_details.tmpl
+++ b/user_settings_appearance.tmpl
@@ -1,8 +1,12 @@
-{{if .User.Language}}
-       <div class="flex-item-body">
-               <span class="flex-text-inline">
-                       <b>{{ctx.Locale.Tr "settings.language"}}:</b>
-                       {{range .AllLangs}}{{if eq $.User.Language .Lang}}{{.Name}}{{end}}{{end}}
-               </span>
+<div class="field">
+       <div class="ui language selection dropdown" id="language">
+               <input name="language" type="hidden" value="{{.SignedUser.Language}}">
+               {{svg "octicon-triangle-down" 14 "dropdown icon"}}
+               <div class="text">{{range .AllLangs}}{{if eq $.SignedUser.Language .Lang}}{{.Name}}{{end}}{{end}}</div>
+               <div class="menu">
+                       {{range .AllLangs}}
+                               <div class="item{{if eq $.SignedUser.Language .Lang}} active selected{{end}}" data-value="{{.Lang}}">{{.Name}}</div>
+                       {{end}}
+               </div>
        </div>
-{{end}}
+</div>
```
i don't see a common template here ?!?